pages/docs/data-ops/dbt-integration.mdx

```diff
@@ -18,25 +18,36 @@ Hashboard can import models directly from dbt by specifying measures in your dbt
 
 <Steps>
 
+### Configure your `.hbproject` file
+
+If you have not already, configure the root of your Hashboard project and your dbt project with [`hb init`](/docs/data-ops/cli/v3/init).
+
+Set the `dbtRoot` key in your Hashboard project file using [`hb init --dbt-root=path/to/dbt/project`](/docs/data-ops/cli/v3/init) or manually editing the `.hbproject` file if it already exists.
+
+```yaml filename="example .hbproject file" copy
+# This file marks the root of your Hashboard project directory. It determines
+# the identity of the resources in your local filesystem
+# and encodes additional project configuration metadata.
+# For additional information please visit docs.hashboard.com.
+dbtRoot: ./dbt/
+```
+
 ### Configure your `dbt_project.yml`
 
-You can use the `hashboard-defaults` key to specify the hashboard version (which is required) and connection information.
+You can use the `hashboard-defaults` key to specify the hashboard version and the data connection to use.
 
 ```yaml filename="dbt_project.yml" copy
 models:
   +meta:
     # hashboard-defaults settings will be merged into each Hashboard model
     hashboard-defaults:
       hbVersion: "1.0"
-      # preventUpdatesFromUI defaults to false, but can be set to true
-      preventUpdatesFromUI: true
-      source:
-        connectionName: bigquery-prod # the name of the conneciton in Hashboard
+      connectionName: # which Hashboard data connection to use
 ```
 
 ### Define model information
 
-The simplest example of a Hashboard model just defines a measure. Hashboard models must include the `hashboard` key in the dbt model.
+The simplest example of a Hashboard model just defines a measure. Hashboard models must include the `hashboard` key in the dbt model meta.
```

Comment:
Might reword to 'Only dbt models with the `hashboard` key present in the meta field will be built by the integration' for clarity?